pkg/controller/servicebinding/reconciler_test.go

```diff
@@ -447,6 +520,33 @@ func TestEmptyApplication(t *testing.T) {
 
 	requireConditionPresentAndTrue(t, CollectionReady, sbrOutput.Status.Conditions)
 	requireConditionPresentAndFalse(t, InjectionReady, sbrOutput.Status.Conditions)
+	requireConditionPresentAndTrue(t, BindingReady, sbrOutput.Status.Conditions)
+}
+
+// TestEmptyServiceSelector tests that Status is successfully updated when ServiceSelector is empty
+func TestEmptyServiceSelector(t *testing.T) {
```

Comment:
please rename this function to reflect what the comment above says.

Reply:
@pedjak 
What name should it be instead of TestEmptyServiceSelector?

Reply:
the comment line above has a few hints how the test function name could look like. It should contain the context (empty selector) and the consequence (status is successfully updated)